### ISSUE TITLE
Change height of x-logo in yaml

### ISSUE THREAD

**Issue by SrikrishnaCodeNebula** [Redocly/redoc#2556]
I have added url of x-logo in yaml file. But want to change height and width of the x-logo and I have tried introducing maxHeight and minHeight properties but it is not working.

**Comment by lornajane** [comment]
Hi @SrikrishnaCodeNebula you can find the documentation for the `x-logo` feature here: https://redocly.com/docs/redoc/redoc-vendor-extensions#x-logo

The page includes information about the supported options.

**Comment by SrikrishnaCodeNebula** [comment]
Hi @lornajane But it does not have any options for height and width as well. It has options to add url, backgroundcolor, href etc. Can you please let me know how to add height and width to x-logo?

**Comment by lornajane** [comment]
We don't have configurable options for resizing the image in the docs page, could you try resizing it before you use it as the logo?

**Comment by SrikrishnaCodeNebula** [comment]
@lornajane you mean to reduce the width and height of the image beforehand and then using it in x-logo url?

**Comment by SoraSabat** [comment]
@SrikrishnaCodeNebula Yes that's what she meant. Use something like Photoshop or illustrator to change dimensions of the image and then use it in the url.

**Comment by AlexVarchuk** [comment]
Please use the prepared image for `x-logo`. It should work.
If you have additional questions please reopen the issue
